.github/workflows/therock-ci.yml

```diff
@@ -0,0 +1,55 @@
+name: TheRock CI for composable_kernel
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
```

Comment:
With regards to the build time, we might need to bring those down first (?) or switch to scheduled.